src/main/resources/application.conf

```diff
@@ -9,4 +9,6 @@ dynamodb {
   referenceCounter = ${?REFERENCE_COUNTER}
   queryParam = "numberofrefs"
   queryParam = ${?QUERY_PARAM}
+  referenceLimit = 5000
+  referenceLimit = ${?REFERENCE_LIMIT}
```

Comment:
Should add `REFERENCE_LIMIT` as an environmental variable to the Lambda in the Terraform.